invenio_records_resources/services/files/schema.py

```diff
@@ -53,7 +53,7 @@ class Meta:
 
         unknown = INCLUDE
 
-    key = SanitizedUnicode(required=True)
+    key = Str(required=True)
```

Comment:
ping @slint 
I am not sure if this is an ultimate solution, the filename is still a kind of user input, I would expect to sanitize it